o_el_iks.API/Program.cs

```diff
@@ -59,4 +116,39 @@ public class RegistrationData
 {
     public string email { get; set; }
     public string password { get; set; }
+}
+
+public class LoggingData
```

Comment:
Jak już pewnie zauważyłeś (a przynajmniej na razie taki był cel robienia wszystkich zadań w kontrollerze) ta klasa strasznie nam się rozrasta i staje się nieczytelna. To pora żeby zacząć wydzielać sobie te klasy do osobnych plików